### PR TITLE
Feature: Persian language Support

### PR DESCRIPTION
**Pull request by CodeWithEmad**
**TL;DR -** [ A short summary of what this PR does and why ]

Hello everyone
I'm glad to announce that we translated all openedx components in transifex to Persian and fa_IR is 100% translated and 100% reviewed.

**What changed?**

in this PR, I've added:
new translations to `i18n/messages/fa_IR.json`,
`fa-ir` in messages in `src/i18n/index.js`,
and `fa_IR` code in `transifex_langs` variable in `Makefile`.

is there anything else I should do? 

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad - apologies for the delay on this. I will check in on getting this reviewed. In the meantime, would you mind resolving the branch conflicts?

CC: @openedx/content-aurora @mattcarter

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad! Would you mind resolving the branch conflicts?

CC: @mattcarter @openedx/content-aurora for merging (as owners of the repo). Thanks!

**Comment by CodeWithEmad** [comment]
Hi @mphilbrick211.
sorry for the delay. all conflicts resolved.

**Comment by CodeWithEmad** [comment]
also @mphilbrick211 
I've added persian langauge support to frontend

**Comment by CodeWithEmad** [comment]
@mphilbrick211 in the https://github.com/openedx/frontend-platform/pull/385, I've added Persian language support to the [frontend-platform](https://github.com/openedx/frontend-platform) repo also.
but looks like they forgot to review/merge it.
that PR was created in October 2022! :)

**Comment by e0d** [comment]
@CodeWithEmad I've run the tests and there's a commit lint issue:
```
⧗   input: reformat: reorder imports alphabetically
✖   type must be one of [revert, feat, fix, perf, docs, test, build, refactor, style, chore, temp] [type-enum]
```

Can this PR merge prior to https://github.com/openedx/frontend-platform/pull/385?  I realize it won't work, but was curious if it would break anything.

**Comment by CodeWithEmad** [comment]
> @CodeWithEmad I've run the tests and there's a commit lint issue:

oops. my bad. commit type changed to `refactor`.



> Can this PR merge prior to [openedx/frontend-platform#385](https://github.com/openedx/frontend-platform/pull/385)? I realize it won't work, but was curious if it would break anything.

No, it won't break anything. the only issue here is that the `frontend platform` doesn't recognize `fa-ir` code as an RTL language, and it will show the translations from Left to Right.

**Comment by e0d** [comment]
@CodeWithEmad excellent, all green now.

**Review by jansenk — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@CodeWithEmad 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-gradebook&entry.867055334=https://github.com/openedx/frontend-app-gradebook/pull/279&entry.1484655318=https://github.com/CodeWithEmad&entry.752974735=2022-10-17+07:50&entry.1917517419=2023-01-23+15:22&entry.2133058324=Yes) so we can improve your experience in the future.
